test/cypress/integration/widgets/deployment_button_spec.ts

```diff
@@ -520,5 +558,57 @@ describe('Create Deployment Button widget', () => {
                 verifyTextInput('null');
             });
         });
+
+        it('textarea', () => {
+            selectBlueprintInModal('textarea');
+
+            cy.contains('.field', 'textarea_default_en').within(() => {
+                verifyTextareaContent('en');
+            });
+
+            cy.contains('.field', 'textarea_display_rows_5').within(() => {
+                verifyTextareaContent('');
+            });
+
+            cy.contains('.field', 'textarea_default_en').within(() => {
+                verifyTextareaRows();
+            });
+
+            cy.contains('.field', 'textarea_display_rows_5').within(() => {
+                verifyTextareaRows(5);
+            });
+        });
+
+        it('blueprint_id', () => {
+            selectBlueprintInModal('blueprint_id');
+
+            cy.contains('.field', 'blueprint_id_all').within(() => {
+                verifyNumberOfOptions(5, true);
+            });
+
+            cy.contains('.field', 'blueprint_id_name_contains_blueprint_id_type').within(() => {
+                verifyNumberOfOptions(1);
+            });
+        });
+
+        it('deployment_id', () => {
+            const testDeploymentId = `${emptyBlueprintId}deployment1234`;
+
+            cy.deployBlueprint(emptyBlueprintId, testDeploymentId);
```

Comment:
As `emptyBluepirintId` is used only here, maybe worth to move uploading that blueprint as well? That way `before` step will be shorter and more precisely defined.
```suggestion
            cy.uploadBlueprint('blueprints/empty.zip', emptyBlueprintId)
              .deployBlueprint(emptyBlueprintId, testDeploymentId);
```